tests/baselines/reference/moduleExportAliasElementAccessExpression.js

```diff
@@ -0,0 +1,12 @@
+//// [moduleExportAliasElementAccessExpression.js]
+function D () { }
+exports["D"] = D;
+
+
+//// [moduleExportAliasElementAccessExpression.js]
+function D() { }
+exports["D"] = D;

```

Comment:
And non-identifier names, like `"A B"`, emit as...what? Probably needs a test.

Reply:
_A_B. I added a test the in the bug, but I guess I can add it here.